heareval/predictions/task_predictions.py

```diff
@@ -25,15 +25,20 @@
 import pandas as pd
 import pytorch_lightning as pl
 import torch
+import wandb
```

Comment:
What does wandb do? Have we added to the setup.py requirements?

Reply:
I see it was already imported below so I guess this was added in a previous PR :)

Reply:
@jorshi better than tensorboard and automatically logs a lot of performance metrics like disk writes, GPU memory transfer, etc.